pkg/http/http_test.go

```diff
@@ -57,11 +58,13 @@ func (suite *TestHttpServerSuite) TestServer() {
 	wg.Add(1)
 	go remediator_http.NewServer(suite.logger).Serve(ctx, &wg)
 
+	time.Sleep(10 * time.Second)
```

Comment:
prefer 0.1 or so, it should be fast

Reply:
changed to 1sec

Reply:
prefer even lower if it works ... keeping tests fast is nice :)